learntools/machine_learning/nbs/tut3.ipynb

```diff
@@ -0,0 +1,261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "dc1e7b02-f022-48ca-a270-21f4b0c70835",
+    "_uuid": "0696f41450f17b2e0ebbbbb30a29053cde7de211"
+   },
+   "source": [
+    "**[Machine Learning Course Home Page](kaggle.com/learn/machine-learning).**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Selecting and Data for Modeling\n",
+    "Your dataset had  too many variables to wrap your head around, or even to print out nicely.  How can you pare down this overwhelming amount of data to something you can understand?\n",
+    "\n",
+    "To show you the techniques, we'll start by picking a few variables using our intuition. Later tutorials will show you statistical techniques to  automatically prioritize variables.\n",
+    "\n",
+    "To choose variables/columns, we'll need to see a list of all columns in the dataset. That is done with the **columns** property of the DataFrame (the bottom line of code below).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "melbourne_file_path = '../input/melbourne-housing-snapshot/melb_data.csv'\n",
+    "melbourne_data = pd.read_csv(melbourne_file_path) \n",
+    "print(melbourne_data.columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The Melbourne data has somemissing values (some houses for which some variables weren't recorded.)\n",
+    "# We'll learn to handle missing values in a later tutorial.  \n",
+    "# Your Iowa data doesn't have missing values in the predictors you use. \n",
+    "# So we will take the simplest option for now, and drop those houses from our data. \n",
+    "#Don't worry about this much for now, though the code is:\n",
```

Comment:
If you think this really isn't important, what about just hiding the code cell?

Reply:
Planning to hide it, and had forgotten to do that. 

Additionally I'm leaving this comment so that it doesn't seem like I'm doing sleight-of-hand to anyone who opens up the hidden cell.